content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Sending large volumes of metrics
+kind: documentation
+description: "Optimizing DogStatsD for high throughput"
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: "Introduction to DogStatsD"
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: "Official and Community-contributed API and DogStatsD client libraries"
+- link: "https://github.com/DataDog/datadog-agent/tree/master/pkg/dogstatsd"
+  tag: "GitHub"
+  text: "DogStatsD source code"
+---
+
+DogStatsD works by sending metrics generated from your application to the [agent](https://docs.datadoghq.com/agent/) over a transport protocol. This transport protocol can be either UDP (User Datagram Protocol) or UDS (Unix Domain Socket).
+
+When DogStatsD is used to send a large volume of metrics to a single Agent, if proper measures are not taken, it is common to end up with the following symptoms:
+
+- High agent CPU usage
+- Dropped datagrams / metrics
+- (UDS) DogStatsD client library returning lots of errors
+
+Most of the time those symptoms can be alleviated by tweaking some configuration options described below.
+
+## General tips
+
+### Enable buffering on your client
+
+Some StatsD and DogStatsD clients, by default, will send one metric per datagram. This adds considerable overhead on the client, the operating system, and the Agent. If your client supports buffering multiple metrics in one datagram, enabling this option could bring noticeable improvements.
+
+Here are a few examples for supported clients:
+
+Please refer to your client documentation for additional details.
```

Comment:
I'd put this after the tabs, seems more natural (I expect the examples to be right after the sentence which introduces them)